### ISSUE TITLE
CMS_TOOLBAR_HIDE causes 'WSGIRequest' object has no attribute 'toolbar'

### ISSUE THREAD

**Issue by darbula** [divio/django-cms#4537]
When `CMS_TOOLBAR_HIDE = True` then saving plugin causes `'WSGIRequest' object has no attribute 'toolbar'` `AttributeError` in `pageadmin.py`.

In `pageadmin.py`this:

```py
if not request.toolbar
```

should probably be replaced with:

```py
if not getattr(request, 'toolbar', False)
```

**Comment by FinalAngel** [comment]
@darbula is this still the case with django-cms 3.2?

**Comment by darbula** [comment]
Sorry for the delay, after upgrading to develop I get following error https://github.com/divio/django-cms/issues/4666 so I cannot confirm if this issue is resolved.

**Comment by mkoistinen** [comment]
I'm unable to reproduce this, per se. It would appear that in v3.2.0, this setting has no effect at all (toolbar still appears when `CMS_TOOLBAR_HIDE = True`).

**Comment by evildmp** [comment]
Then we should have documented that it was removed!

**Comment by mkoistinen** [comment]
For posterity, the PR that introduced this fix/feature was committed into the 3.1.1 release as #4056. There are good tests that still run and pass, so, perhaps there's something odd happening locally for me. None-the-less, I am fixing this specific issue.

**Comment by mkoistinen** [comment]
Fixed by #4854.
